tests/EdFi.Security.DataAccess.UnitTests/Repositories/CachedSecurityRepositoryTests.cs

```diff
@@ -46,35 +46,35 @@ public void Should_throw_argument_exception(int cacheTimeoutInMinutes)
             }
         }
 
-        public class When_executing_decorated_method_with_ten_minute_cache_expiration_but_five_minutes_have_passed : CachedSecurityRepositoryTests
```

Comment:
These tests are removed because mocking EF Core with the completeness and accuracy necessary to test the caching logic would be difficult and risky. A better approach may be to extricate the caching implementation from the repository class, allowing it to be tested without needing to call an EF Core context.